yt/frontends/boxlib/io.py

```diff
@@ -122,6 +122,7 @@ def _read_chunk_data(self, chunk, fields):
                         local_offset = 0
                     else:
                         local_offset += size
+            f.close()
```

Comment:
@mameehan5 wasn't this supposed to be it's own PR ?

Reply:
It's probably OK to include it here if we're aiming to get this PR in soon.

Reply:
sure, but I'd merge this on the spot if it had it's own PR :)

Reply:
Yes, I knew to add it as a separate PR but I was testing first on this branch. Updating this branch is still on my todo list but I am not sure how soon it will get done. I added a new PR for just the close statement though! #3515